components/RestPasswordForm/ResetPasswordForm.tsx

```diff
@@ -0,0 +1,192 @@
+// Copyright (c) Gridiron Survivor.
+// Licensed under the MIT License.
+
+'use client';
+import { resetPassword } from '@/api/apiFunctions';
+import Alert from '@/components/AlertNotification/AlertNotification';
+import { AlertVariants } from '@/components/AlertNotification/Alerts.enum';
+import { Button } from '@/components/Button/Button';
+import { Input } from '@/components/Input/Input';
+import LoadingSpinner from '@/components/LoadingSpinner/LoadingSpinner';
+import { zodResolver } from '@hookform/resolvers/zod';
+import { JSX, useState } from 'react';
+import { Control, SubmitHandler, useForm, useWatch } from 'react-hook-form';
+import toast from 'react-hot-toast';
+import { z } from 'zod';
+import {
+  Form,
+  FormControl,
+  FormField,
+  FormItem,
+  FormLabel,
+  FormMessage,
+} from '../Form/Form';
+import { useAuthContext } from '@/context/AuthContextProvider';
```

Comment:
nit: these imports are not sorted alphabetically.

Reply:
✅ fixed with SHIFT-OPTION-O